### DISCUSSION TITLE
Misquote on Documentation Relating to On-Campus Crimes

### DISCUSSION THREAD

**Discussion by JorgeRivera94** [uprm-inso4116-2024-2025-s1/semester-project-SafeRUM#247] [Q&A] [answered]
The professor left feedback on Page 28 on our documentation states "Given that the National Center for Education Statistics [reported](https://nces.ed.gov/fastfacts/display.asp?id=804) 16.9 total on-campus crimes in 2021, it is very important that said issues with the current emergency communication system get addressed." His comment questions the 16.9 number asking "What would .9 crimes be?"

Reading the linked source, this figure is incomplete in our documentation. The figure taken from the statement "This translates to 16.9 on-campus crimes reported per 10,000 full-time-equivalent (FTE) students enrolled." should be quoted directly from the source.

I am already going over UPRM criminal acts reports for task #177. Should I update this 16.9 figure with UPRM's reported crimes for 2024 or just update the quote?

@lex939

**Comment by lex939** [comment, marked as answer]
I would update the documentation to quote the original source accurately as 'Approximately 16.9 on-campus crimes reported per 10,000 FTE students enrolled' to address the professor's comment. Make sure to refer directly to the sources when updating the figure. Additionally, since you are reviewing UPRM's 2024 crime data for task #177, you could include that as a contrasting figure for a more specific comparison to enhance the documentation.
